deploy/determined_deploy/local/cli.py

```diff
@@ -204,16 +219,20 @@ def handle_agent_up(args):
         version=args.det_version,
         labels=None,
         autorestart=(not args.no_autorestart),
+        cluster_name=args.cluster_name,
     )
 
 
-def handle_agent_down(args):
+def handle_agent_down(args: argparse.Namespace) -> None:
     if args.all:
         cluster_utils.stop_all_agents()
     else:
         cluster_utils.stop_agent(agent_name=args.agent_name)
 
 
+HandlerFn = Callable[[argparse.Namespace], None]
```

Comment:
non-blocking: This is short enough that I don't see the need to give it a name for the one usage, but your call.

Reply:
Yeah... definitely.  Nothing to see here.